client/state/initial-state.js

```diff
@@ -77,7 +78,7 @@ export function persistOnChange( reduxStore, serializeState = serialize ) {
 			.catch( ( setError ) => {
 				debug( 'failed to set redux-store state', setError );
 			} );
-	} );
+	}, SERIALIZE_THROTTLE, { leading: false, trailing: true } ) );
```

Comment:
`trailing` [defaults to `true`](https://lodash.com/docs#throttle), but I guess it doesn't hurt to be explicit.